config/projects/datadog-agent.rb

```diff
@@ -193,26 +185,6 @@
 dependency 'psutil'
 dependency 'requests'
 
-unless ENV["USE_INTEGRATION_SDK"] == 'true'
```

Comment:
Can you keep this flag for now? (i.e. `dependency 'datadog-agent-integration-deps'`)